### PR TITLE
Translations

### PR DESCRIPTION
**Pull request by yorikvanhavre**
Your usual dose of crowdin translations

### CODE REVIEW

File: src/Base/Resources/translations/Base_zh-CN.ts [hunk 1]
```diff
@@ -26,27 +26,27 @@
...
       <location filename="../../UnitsApi.cpp" line="79"/>
       <source>Building US (ft-in, sqft, cft)</source>
-      <translation type="unfinished">Building US (ft-in, sqft, cft)</translation>
+      <translation>美制 (ft-in/sqft/cuft)</translation>
     </message>
     <message>
       <location filename="../../UnitsApi.cpp" line="81"/>
       <source>Metric small parts &amp; CNC (mm, mm/min)</source>
-      <translation type="unfinished">Metric small parts &amp; CNC (mm, mm/min)</translation>
+      <translation>公制 (㎜, ㎧)</translation>
```
[the remark this answers is not in the document]
Comment (yorikvanhavre) [reply src/Base/Resources/translations/Base_zh-CN.ts:39]: Wrong translation. It happens unfortunately often... It needs to be notified on crowdin and people there need to fix it

[the remark this answers is not in the document]
Comment (hyarion) [reply src/Base/Resources/translations/Base_zh-CN.ts:39]: It's now reported in crowdin.

[the remark this answers is not in the document]
Comment (yorikvanhavre) [reply src/Base/Resources/translations/Base_zh-CN.ts:39]: thanks!

**Comment by yorikvanhavre** [comment]
backported

**Pull request merged**